CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### Changed
+
+- **Breaking!** Changed `register()` to work with the new device registration flow. `register()` now return device registration information, but not the full device object.
```

Comment:
It's good to mention the format at least here, like `device registration information ({ ... })`